tslib/react/src/utils/loadStorageFromFile.ts

```diff
@@ -0,0 +1,45 @@
+import { JournalFileStorage, SQLite3Storage } from "@optuna/storage"
```

Comment:
Could you move this file into `tslib/storage` or just leaving it in the `standalone_app`?